### PR TITLE
Port 7533 Update wiz docs to show information on config params

### PR DESCRIPTION
**Pull request by phalbert**
# Description

- Fix variable declaration in Azure Devops pipeline from `${VARIABLE_NAME}` to `$(VARIABLE_NAME)`
- Add guide on setting variable groups.
- Add comment on how to get Wiz credentials

## Updated docs pages

Please also include the path for the updated docs

- Blueprint (`/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 1]
```diff
@@ -18,9 +19,21 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
 
-<Prerequisites />
+1.**Port Credentials**:
 
-Your Wiz credentials should have the `read:projects` and `read:issues` permission scopes. Visit the Wiz [documentation](https://integrate.wiz.io/reference/prerequisites) for a guide on how to get your credentials as well as set permissions.
+<FindCredentials />
+
+2. **Wiz Credentials**:
+
+:::info GETTING YOUR WIZ CREDENTIALS
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:28]: @phalbert sending an email in order to get your client ID and secret seems strange. Here is a doc page of another product that explains how to acquire them:
https://docs.safe.security/docs/wizio#42-client-id-and-client-secret

Please change the instructions to these.

**Review by hadar-co — CHANGES_REQUESTED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md (hunk 1 above)
Comment (phalbert) [reply docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md]: Thanks a lot for this link. I have revised the instructions now.

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 2]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+
+<br />
+
+1. **Finding Your Wiz API URL**:
+    - Log in to Wiz account.
+    - Click the User Profile icon (top right) and select "User Settings."
+    - Click "Tenant" in the left menu.
+    - Copy the API Endpoint URL: It will be displayed on the screen.
+
+<img src='/img/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wizApi.png' width='85%' border='1px' />
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:51]: This image is wrong, take a better look at the docs @hadar-co linked, the image that should be here is different.

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 3]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+
+For more details, refer to the [documentation](https://docs.wiz.io/wiz-docs/docs/using-the-wiz-api#the-graphql-endpoint)
+
+<br />
+
+2. **Getting the Client ID and Client Secret**
+
+You should use a Wiz account with Project Admin role to complete this step.
+
+    - Log in to Wiz: Access your Wiz account.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:61]: This section is missing one of the images from the guide @hadar-co linked

**Review by MPTG94 — CHANGES_REQUESTED** [review]
Please make updates according to my comments

**Comment by phalbert** [comment]
@MPTG94 I have resolved these

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 4]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+- Amazon Cognito:	https://auth.app.wiz.io/oauth/token
+- Auth0:	https://auth.wiz.io/oauth/token
+
+Learn more [here](https://win.wiz.io/reference/quickstart#generate-a-bearer-token-and-start-using-wiz-api).
+:::
+
+<br />
+
+1. **Finding Your Wiz API URL**:
+    - Log in to Wiz account.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:46]:
```suggestion
    - Login to your Wiz account.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 5]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+    - Click the User Profile icon (top right) and select "User Settings."
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:47]:
```suggestion
    - Click the **User Profile** icon available at the top right of the screen and click the **User Settings** option.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 6]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+    - Click "Tenant" in the left menu.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:48]:
```suggestion
    - Click the **Tenant** option from the left options menu.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 7]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+    - Copy the API Endpoint URL: It will be displayed on the screen.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:49]:
```suggestion
    - The system displays the **API Endpoint URL**.
    - Copy and save the **API URL** to use while configuring the Wiz intergration.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 8]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+<img src='/img/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wizApiUrl.png' width='85%' border='1px' />
+
+For more details, refer to the [documentation](https://docs.wiz.io/wiz-docs/docs/using-the-wiz-api#the-graphql-endpoint)
+
+<br />
+
+2. **Getting the Client ID and Client Secret**
+
+You should use a Wiz account with the Project Admin role to complete this step.
+    - Log in to Wiz: Access your Wiz account.
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:60]:
```suggestion
You must create a service account in Wiz to generate the Client ID and Client Secret. Follow the below steps to get the Client ID and Client Secret:
    - Login to **Wiz with the Project Admin role**.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 9]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+    - Open Settings: Click the Settings icon (top-right).
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:61]:
```suggestion
    - Click the **Settings** icon available at the top-right of the page.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 10]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+  <img src='/img/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wizAddSvcAccount.png' width='85%' border='1px' />
+    - Go to Service Accounts: In the left menu, find and select "Service Accounts".
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:63]:
```suggestion
    - On the Settings page, Click **Service Accounts** from the left menu.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 11]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+    - Create a Service Account:
+        - Click "Add Service Account".
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:65]:
```suggestion
        - Click **Add Service Account**.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 12]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+        - Provide a descriptive "Service Account Name".
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:66]:
```suggestion
        - Provide a descriptive **Service Account Name**.
```

File: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md [hunk 13]
```diff
@@ -18,10 +19,68 @@ Our Wiz integration allows you to import `projects`, `issues`, `controls`, and `
...
+        - Type: Select "Custom Integration (GraphQL API)".
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md:67]:
```suggestion
        - Type: Select **Custom Integration (GraphQL API)**.
```

**Review by MPTG94 — CHANGES_REQUESTED** [review]
Please apply my suggestions

**Comment by phalbert** [comment]
@MPTG94, thanks for catching these. They were avoidable. I will be keen.

**Review by MPTG94 — APPROVED** [review]
LGTM

**Pull request merged**